xbmc/music/windows/GUIWindowMusicNav.cpp

```diff
@@ -340,7 +340,8 @@ bool CGUIWindowMusicNav::GetDirectory(const std::string &strDirectory, CFileItem
     else
       items.SetContent("");
   }
-  else if (items.IsPlayList())
+  else if (items.IsPlayList() ||
+          (items.IsSmartPlayList() && items.GetContent() == "mixed"))
     items.SetContent("songs");
```

Comment:
Yes. We can also leave it at mixed if that's better for skins. But right now it's "files" which is wrong.